### ISSUE TITLE
Error initializing Discord application commands: Command file not found: Verify.json

### ISSUE THREAD

**Issue by CuriousNezy** [IOL-Haven/discord-verifier#4]
Bot is online while server is online. No additional configs .jsons created after verify. No structure of command creation...
![image](https://github.com/user-attachments/assets/d7329256-0def-4652-a14c-8ab71199f62f)

Installed latest version. I do have Fabric API on. Doesn't create anything else even after adding bot token. But bot does go online when booting up the server.

**Comment by TechnoPorg** [comment]
Thank you for the issue report! I believe I know the root cause, and I will get on fixing it as soon as possible.

**Comment by CuriousNezy** [comment]
Let me know if you need any other conditions or information, appreciate you taking a looking into this!!

**Comment by TechnoPorg** [comment]
I fixed the bug via #5, and then another small issue that I caught as well.
I have now tested the mod more extensively outside of my development environment. Hopefully this means all problems with the current version have now been caught, as doing so earlier would have allowed me to notice this issue before releasing the mod.

Jars to test the latest version are available here: https://github.com/IOL-Haven/discord-verifier/actions/runs/10982217247/artifacts/1963364325
